react/components/EditorContainer/Sidebar/ComponentList/utils.ts

```diff
@@ -17,21 +19,15 @@ export const normalizeComponents = (components: SidebarComponent[]) =>
             }
             : item,
         ),
-      components
-        .filter(isTopLevelComponent)
-        .map(defineSortability) as NormalizedComponent[],
+      roots as NormalizedComponent[]
     )
+    .map(item => (
+      {...item, isSortable: true}
```

Comment:
If all normalized components are now sortable, I believe there's need for that property anymore.

Reply:
Now leaves are not sortable and roots are sortable, as before.